src/lseq.rs

```diff
@@ -279,7 +272,10 @@ impl<T: Clone, A: Actor> CmRDT for LSeq<T, A> {
     ///
     /// If the operation is a delete and the identifier is **not** present in the LSEQ instance the
     /// result is a no-op
+    ///
+    /// If the op's dot is less than current Vector Clock dot for that actor, the result is a no-op
```

Comment:
Not quite true with our implementation here, but we can resolve this later when we work through  idempotency

Reply:
truetrue after the rework there. I'll update it